docs/l2-adjacent-mode.rst

```diff
@@ -21,6 +21,12 @@ L2-adjacent mode is the **default mode of operation** for the |agent-short|.
    Many L3 segmentation mode parameters depend on other configuration parameters.
    Read about the :ref:`F5 agent configuration parameters <agent-config-parameters>` before changing these settings to ensure they don't conflict.
 
+**L2 Population Service**
+
+The OpenStack Neutron ML2 L2 population service is supported by the F5 LBaaS agent. When you enable L2 population, the agent registers for Neutron L2 population updates and populates tunnel FDB entries in your BIG-IP. When VIPs are placed on tenant overlay networks, the F5 LBaaS agent sends tunnel update messages to Open vSwitch agents informing them of TMOS device VTEPs. This enables tenant guest virtual machines or network node services to interact with the TMOS provisioned VIPs across overlay networks. An F5 LBaaS agent's connected TMOS VTEP addresses are placed in the agent's configurations and reported to Neutron. Without L2 population enabled, pool member migration will not function properly. You should enable this service if you migrate pool members to different virtual machines without re-creating your load balancer configuration. To enable the L2 population service in the F5 LBaaS agent, edit the agent configuration file, ``/etc/neutron/services/f5/f5-openstack-agent.ini``, and set ``l2_population = True``.
```

Comment:
Change the first sentence to "The F5 LBaaS agent supports the OpenStack Neutron ML2 population service."

Reply:
- "BIG-IP" --> "BIG-IP device"
- "When VIPs are placed" -> "When you place VIPs"
- "Open vSwitch agents," (add comma)
- "An F5 LBaaS agent's connected TMOS VTEP addresses are placed in the agent's configurations and reported to Neutron." --> "The F5 LBaaS Agent reports the BIG-IP VTEP addresses stored in its configuration to Neutron."
(insert line break)
- "Enable L2 population if you intend to migrate pool members to different virtual machines without re-creating your load balancer configuration. Pool member migration won't function properly if L2 population isn't enabled."
(insert line break)

Reply:
Fixed.